src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -80,7 +88,21 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
                 await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
 
                 if (paymentData && isVaultedInstrument(paymentData)) {
-                    return this._store.dispatch(this._paymentActionCreator.submitPayment(payment));
+                    if (this._isHostedPaymentFormEnabled(payment.methodId, payment.gatewayId) && this._shouldRenderHostedForm) {
+                        const form = this._hostedForm;
+
+                        if (!form) {
+                            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+                        }
+
+                        try {
+                            await form.validate();
+
+                            return this._store.dispatch(this._paymentActionCreator.submitPayment(payment));
+                        } catch (error) {
+                            throw new Error(error.message);
```

Comment:
this seems to simply rethrow the same error. Do you ned this try/catch block at all?

Reply:
Yes, for some reason (i think its because the type of the Error) it just console log the error it doesnt throw it!

Reply:
its for form.validate error

Reply:
We don't do this for other strategies, e.g. `credit-card-payment-strategy` so I'm still not convinced why it's needed here.